src/aqua/lra_generator/lra_generator.py

```diff
@@ -39,11 +40,15 @@ def __init__(self,
                  resolution=None, frequency=None, fix=True,
                  outdir=None, tmpdir=None, nproc=1,
                  loglevel=None,
-                 region=None,
+                 region=None, drop=False,
                  overwrite=False, definitive=False,
                  performance_reporting=False,
                  rebuild=False,
-                 exclude_incomplete=False, **kwargs):
+                 exclude_incomplete=False,
+                 stat="mean",
+                 compact="xarray",
+                 cdo_options=["-f", "nc4", "-z", "zip_1"],
+                 **kwargs):
```

Comment:
what is `zip_1`? why not simply a`zip`?

Reply:
zip_1 is zip compression level 1
We use the same option when writing with xarray

Reply:
I suspect it is the same of using `zip`! but of course no hassle, go on and merge for me